docs/streams/developer-guide/config-streams.html

```diff
@@ -600,8 +613,54 @@ <h4><a class="toc-backref" href="#id33">default.windowed.value.serde.inner</a><a
           <span id="streams-developer-guide-max-task-idle-ms"></span><h4><a class="toc-backref" href="#id28">max.task.idle.ms</a><a class="headerlink" href="#max-task-idle-ms" title="Permalink to this headline"></a></h4>
           <blockquote>
             <div>
-              The maximum amount of time a task will idle without processing data when waiting for all of its input partition buffers to contain records. This can help avoid potential out-of-order
-              processing when the task has multiple input streams, as in a join, for example. Setting this to a nonzero value may increase latency but will improve time synchronization.
+              <p>
+                This configuration controls how long Streams will wait to fetch data in order to
+                provide in-order processing semantics.
+              </p>
+              <p>
+                When processing a task has multiple input partitions (as in a join or merge),
```

Comment:
When processing a task **has** multiple input partitions -> When processing a task **having** multiple input partitions
Is that right?

Reply:
Ah, good catch. This was a casualty of refactoring. This is what I meant to say:

```suggestion
                When processing a task that has multiple input partitions (as in a join or merge),
```